optuna/terminator/improvement/evaluator.py

```diff
@@ -41,12 +52,10 @@ def evaluate(
 class RegretBoundEvaluator(BaseImprovementEvaluator):
```

Comment:
The `add_random_inputs` argument of `get_preprocessing` is no longer necessary. Could you remove it?

```python
def get_preprocessing(self, add_random_inputs: bool = False) -> BasePreprocessing:
```

Sorry for my commenting here as I cannot directly comment to the line due to the GitHub UI.